python/cisa_risk_meter/build_cisa_risk_meter.py

```diff
@@ -0,0 +1,335 @@
+import os
+import sys
+import time
+import json
+import logging
+import requests
+
+# Humble constants.
+API_MAX_PAGES = 20
+CISA_CUSTOM_FIELD_NAME = "CISA"
+CISA_RISK_METER_NAME = "CISA Exploited Vulnerabilities"
+VULN_UPDATE_LIMIT = 5000
+
+# Maximum page size
+SEARCH_PAGE_SIZE = 5000
+RISK_METER_PAGE_SIZE = 100
+
+# This might have to changed depending on your setup.
+KENNA_BASE_URL = "https://api.kennasecurity.com/"
+
+# Dump JSON in a pretty format.  Great for debugging.
+def print_json(json_obj):
+    print(json.dumps(json_obj, sort_keys=True, indent=2))
+
+# Get the CISA catalog.
+def get_cisa_catalog():
+    get_cisa_catalog_url = f"https://www.cisa.gov/sites/default/files/feeds/known_exploited_vulnerabilities.json"
+    headers = { 'Accept': 'application/json',
+                'Content-Type': 'application/json; charset=utf-8' }
+
+    response = requests.get(get_cisa_catalog_url, headers=headers)
+    if response.status_code != 200:
+        print(f"List CISA Catalog Error: {response.status_code} with {get_cisa_catalog_url}")
+        logging.error(f"List CISA Catalog API, {get_cisa_catalog_url} error: {resp_json}")
+        sys.exit(1)
+
+    resp_json = response.json()
```

Comment:
probably could just return `response.json()` instead of setting it in a new variable.

Reply:
That's a good one, but I think I'll leave it for not, because I would have to go through the blog and renumber most of the code samples.

Reply:
oh geez thats super annoying. ya forget it then haha